### PR TITLE
add: avatar.html and avatar.css component

### PR DESCRIPTION
**Pull request by m26unkwn**
Just created the pull request for the avatar component

Please do give reviews and suggestions.

Screenshots

image

### CODE REVIEW

**Comment by deekshithmd** [comment]
Looks great, variable naming is super simplified and code structure also looks good

**Comment by Mohitkumar33** [comment]
Monu everything is good, the structure and all. But you can do is you can provide light blue colors in some of the places to get the attention of the users. Overall your component library looks nice.

File: components/avatar.html
Comment (PuneetP16) [inline components/avatar.html:174]: Everything looks good just a tip.

Create an account on carbon and then generate an iframe link you will get a shorter iframe source url.

**Review by PuneetP16 — APPROVED** [review]
Good to merge after shortening the iframe source URL.

File: components/avatar.html
Comment (shivampandey0) [inline components/avatar.html:126]: Everything looks good. 
Just the images are broken on the preview url.

**Review by shivampandey0 — COMMENTED** [review]
looks good.

File: components/avatar.html
Comment (m26unkwn) [reply components/avatar.html:126]: Actually, my all assets are in my dev branch that's why it's happening like this. After merging, it will be fine.

But thanks for this review man.

File: components/avatar.html
Comment (m26unkwn) [reply components/avatar.html:174]: Thanks, Puneet, I did use the short link but there is some backend problem in carbon.sh so that's why I opt-out of this.

**Pull request merged**
